apps/OpenSpace/ext/launcher/src/launcherwindow.cpp

```diff
@@ -391,7 +406,19 @@ void LauncherWindow::selectProfile(std::optional<std::string> selection) {
     ghoul_assert(selection.has_value(), "No special item in the profiles");
     if (selection.has_value()) {
         // Having the `if` statement here to satisfy the MSVC code analysis
-        _editProfileButton->setEnabled(std::filesystem::exists(*selection));
+
+        // Enable the Edit button only for the user profiles
+        const bool isUser = selection->starts_with(_userProfilePath.string());
+        _editProfileButton->setEnabled(isUser);
+
+        if (isUser) {
+            _editProfileButton->setToolTip("");
+        }
+        else {
+            _editProfileButton->setToolTip(
+                "Cannot edit the selected profile as it is one of the built-in profiles"
+            );
+        }
```

Comment:
This seems to only be applied after selecting a profile. I can still edit the default profile when starting the launcher, before selecting a profile in the list. 

Please fix :)